### PR TITLE
Tekton Intro Blog

### LINKED ISSUE #15: Deliver Blog:  Tekton, learn about Kabanero's DevOps Toolchain
As part of the August 26th release blog series, we want to introduce Tekton and our Kabanero DevOps Toolchain. The goal is to build momentum through our blog series from August 26 release up to the September events that begin Sept 9th and run through Sept 26.

This blog should describe why we chose Tekton, what it’s good at, how we are using Tekton, the DevOps flow for a team in Kabanero.(webhooks that listen for git events (commit, review, merge).  Describe the different tasks we are building and how Champ can customize build and deploy pipelines.

We should talk about how we're contributing back to the Tekton community and pulling that back into Kabanero.

Given where we're at, I'd like this to be a little forward looking, so need Alan & Steve's views on exactly what we're doing in the very near future ....  introducing an event/notification subsystem to enable a full end-to-end automated Toolchain that can support Tekton pipelines and alternatives like Jenkins.  Replacing webhooks w/ our new eventing subsystem, what we're doing with Razee to tag and drive automated promotion to production clusters, etc.

### PR DESCRIPTION
**Pull request by dewan-ahmed**
closes #15

### CODE REVIEW

**Comment by alohr51** [comment]
@dewan-ahmed is there an issue associated with this blog?

**Comment by anneqm** [comment]
Yes. This is part one of this issue: https://github.com/kabanero-io/blogs/issues/15

File: drafts/2019-11-01-Tekton-Enterprise-CI-CD.adoc [hunk 1]
```diff
@@ -16,37 +16,23 @@ blog_description: "Read about Tekton - a new open source project that provides a
...
-* node-js-express
-* node-js-loopback
-* node-js
+What does it mean to be cloud native and why is it important? Cloud native tools are purpose-built for the cloud. They take advantage of access to instantly available compute and storage resources that can be adapted quickly to the needs of the tool. Cloud native tools also offer simplified maintenance, resiliency, and elasticity, which gives you the ability to scale resources as needed.
 
-Collections are built from https://developers.redhat.com/products/rhel/ubi/[Universal Base Images] and updated to stay current with performance improvements and security currency.  This ensures when applications created with Collections are deployed on OKD or OpenShift the entire stack is supportable with operating system libraries in each container. 
+Introducing Tekton, a new open source project that provides a framework for creating cloud native CI/CD pipelines. Tekton benefits from a robust community of contributors and provides a set of fundamental building blocks for creating a CI/CD environment. Tektonprovides a first-class Kubernetes integration, including custom resource definitions (CRDs) as a standard way to run end-to-end pipelines.
```
Comment (anneqm) [inline drafts/2019-11-01-Tekton-Enterprise-CI-CD.adoc]: Add space in beginning of last sentence `Tektonprovides` should be `Tekton provides`

File: drafts/2019-11-01-Tekton-Enterprise-CI-CD.adoc [hunk 2]
```diff
@@ -16,37 +16,23 @@ blog_description: "Read about Tekton - a new open source project that provides a
...
-
-## Support scripts
-The Kabanero support scripts include a sample script that uninstalls Kabanero and must-gather scripts that collect important diagnostic data.
-
-## Kabanero operator, enhanced
-The Kabanero operator now includes a –targetNameSpaces configuration that defines a list of namespaces that help to manage Appsody application deployments with Kabanero using a controlled set of namespaces defined by the architect and operations roles. To simplify collection development, variable substitution is introduced to allow the same pipeline template to be used for multiple collections. This allows you to use one pipeline for multiple collections.
-
-As always you can, https://kabanero.io/try-it/[try it out]!
+ 
+To see Tekton in action, go to the link:https://kabanero.io/[Kabanero website]. Kabanero brings together Knative, Istio, and Tekton, with Codewind, and Appsody. Together, these open source projects provide an end-to-end solution for you to build, deploy, and manage the lifecycle of Kubernetes-based applications.
```
Comment (anneqm) [inline drafts/2019-11-01-Tekton-Enterprise-CI-CD.adoc]: remove comma after Codewind.

**Review by anneqm — DISMISSED** [review]
Thanks Dewan!  two very minor changes.   Also please double check line spacing around the image on line 33.

File: drafts/2019-11-01-Tekton-Enterprise-CI-CD.adoc (hunk 1 above)
Comment (dewan-ahmed) [reply drafts/2019-11-01-Tekton-Enterprise-CI-CD.adoc]: Done :)

File: drafts/2019-11-01-Tekton-Enterprise-CI-CD.adoc (hunk 2 above)
Comment (dewan-ahmed) [reply drafts/2019-11-01-Tekton-Enterprise-CI-CD.adoc]: Done :)

**Comment by dewan-ahmed** [comment]
Committed with both changes fixed. Not sure about the change with img on line33. Pls let me know and I can re-commit.

File: drafts/2019-11-01-Tekton-Enterprise-CI-CD.adoc [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+A sample Tekton pipeline: 
+
+image::/img/blog/Tekton-Intro.png[link="/img/blogs/Tekton-Intro.png" alt="Tekton Intro"]
+
+
+While Tekton is still new, the community of innovators that are contributing to and using Tekton is growing. Through layered steps and tasks, Tekton pipelines give flexibility to enterprises for their continuous integration and delivery needs. Tekton pipelines also allow enterprises to meet their customer demands by quickly delivering vital application updates that satisfy enterprise-specific quality and security requirements. 
+
+ 
+To see Tekton in action, go to the link:https://kabanero.io/[Kabanero website]. Kabanero brings together Knative, Istio, and Tekton, with Codewind and Appsody. Together, these open source projects provide an end-to-end solution for you to build, deploy, and manage the lifecycle of Kubernetes-based applications.
```
Comment (alohr51) [inline drafts/2019-11-01-Tekton-Enterprise-CI-CD.adoc:39]: I feel like we need a better link. You can't really "see tekton in action" on kabanero.io.

On the other hand, maybe the text can just change to: "To get the latest news with Tekton in Kabanero, check out the..."

Comment (alohr51) [reply drafts/2019-11-01-Tekton-Enterprise-CI-CD.adoc:39]: also keep in mind the blog will be on kabanero.io, so maybe we can just delete that first sentence.

**Review by alohr51 — APPROVED** [review]

**Pull request merged**